databricks/koalas/indexes.py

```diff
@@ -422,6 +446,17 @@ def to_pandas(self) -> pd.MultiIndex:
 
     toPandas = to_pandas
 
+    def dropna(self):
+        """
+        Return MultiIndex without NA/NaN values
+        """
+        kdf = self._kdf
+        sdf = self._kdf._internal.sdf
+        for index_scol in self._kdf._internal.index_scols:
+            sdf = sdf.where(index_scol.isNotNull())
```

Comment:
In fact, can we share the same implementation?

Reply:
okay i'll commit them soon. Thanks for reviewing!!